### ISSUE TITLE
Cannot complete connection to room even though audio is audible

### ISSUE THREAD

**Issue by mattrosemeier2020** [twilio/twilio-video-ios#122]
> Before filing an issue please check that the issue is not already addressed by the following:
>
>  * [Video Guides](https://www.twilio.com/docs/api/video)
>  * [Changelog](https://www.twilio.com/docs/api/video/changelog-twilio-video-ios-version-2x)
> 
> If this is an issue with the SDK itself, file it here. If this is an issue with the QuickStart apps, please use [video-quickstart-ios](https://github.com/twilio/video-quickstart-ios/issues).
>
> Please ensure that you are not sharing any
[Personally Identifiable Information(PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
or sensitive account information (API keys, credentials, etc.) when reporting an issue.

### Description

Downloaded the latest quickstart zip file with the various projects inside.  The VideoQuickStart project itself is not out-of-the-box compatible with the TwilioVideo.framework.  To confirm, this is all from video-quickstart-ios-2.x.  I followed the hand holding XCode does and updated all the calls and class names to the new ones (i.e. XYZ has been renamed to ABC type of stuff)

### Steps to Reproduce

Pull my repo with the updated changes so that the quickstart projects work with the latest TwilioVideoSDK: https://github.com/mattrosemeier2020/TwilioVideoUpdates

#### Code

```swift
see repo
```

#### Expected Behavior

connection successful within a Twilio-provided project

#### Actual Behavior

I am stuck in the "attempting to connect to room" state and never get the `didConnect(to room: Room)` delegate callback.  I can hear audio between the two callers but never get the success message and never see remote video.

#### Reproduces How Often

100%

#### Logs
Debug level logs are helpful when investigating issues. To enable debug level logging, add the following code to your application:

```.swift
TwilioVideoSDK.setLogLevel(.debug)
```

```
// Log output when the issue occurs
```

### Versions

All relevant version information for the issue.

#### Video iOS SDK

13.xxx

#### iOS Device

[e.g. iPhone 8 Plus]

**Comment by ceaglest** [comment]
Hi @mattrosemeier2020,

> Downloaded the latest quickstart zip file with the various projects inside. The VideoQuickStart project itself is not out-of-the-box compatible with the TwilioVideo.framework. To confirm, this is all from video-quickstart-ios-2.x. I followed the hand holding XCode does and updated all the calls and class names to the new ones (i.e. XYZ has been renamed to ABC type of stuff)

The [Quickstart](https://github.com/twilio/video-quickstart-ios) is compatible with our 3.0 SDK, and you do not need to make code changes other than providing credentials. This makes me think you checked out the wrong branch and then tried to convert 2.0 APIs to 3.0, possibly introducing bugs in the process. 

> didConnect(to room: Room)

This is a 2.0 API.

Can you please do the following?

1. Check out the [master branch](https://github.com/twilio/video-quickstart-ios/tree/master/VideoQuickStart)
2. Program in your token server / info from your repo.
3. Connect to a Room.

Let me know if you are still experiencing issues.

Thanks,
Chris

**Comment by mattrosemeier2020** [comment]
Hey @ceaglest thanks for the quick reply.  You are right I was on v2.  I ended up tracing my steps back to figure out how this happened and it looks like `twilio-video-ios/releases/download/2.10.2/TwilioVideo.framework.zip` is linked on the doc landing page as "the latest version."   After doing the pull manually and getting v3 from the changelog It all works great now.  
![Screen Shot 2020-08-17 at 8 54 13 PM](https://user-images.githubusercontent.com/69822965/90458370-3841ad80-e0cc-11ea-8268-e4918f395b7f.png)

**Comment by ceaglest** [comment]
Hey @mattrosemeier2020,

Oh I see. Thanks for pointing out the mistake, I will make sure the docs get updated.

Best,
Chris

**Comment by ceaglest** [comment]
Just to clarify, I looked at the [iOS Getting Started: Manual Instructions](https://www.twilio.com/docs/video/ios-getting-started#manual) page and the link is to 3.6.1 (the latest release). Could you confirm this is the same place that you found the link to the older 2.x version?

**Comment by mattrosemeier2020** [comment]
Yep that's right, and it takes you to this page: https://www.twilio.com/docs/video/ios-v3-getting-started#manual

And underneath the "Manual" section it says 
![Screen Shot 2020-08-18 at 10 06 04 AM](https://user-images.githubusercontent.com/69822965/90523062-75448900-e13a-11ea-82a3-6c1092eae721.png)

And if you click that "Download the Latest" link it takes you to a v2 version it seems.

**Comment by ceaglest** [comment]
Hey @mattrosemeier2020,

> Yep that's right, and it takes you to this page: https://www.twilio.com/docs/video/ios-v3-getting-started#manual

I inspected the source of truth for the markdown in our CMS, and I tried the **Download Latest** Link on a co-worker's machine. In both cases I got:

https://github.com/twilio/twilio-video-ios/releases/download/3.6.1/TwilioVideo.framework.zip

Is it possible there was a caching issue, and cleaning your browser's cache would resolve it? I don't want to take much more of your time with this, so I can continue to investigate internally.

Best,
Chris
